src/test/resources/shutdown_all.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Do you have any gradle experience and can implement the setup and teardown of redis in the gradle build file itself?
That would be nice so we can still run the tests automatically on Jenkins.